src/pages/waiting-room-to-car/waiting-room-to-car.scss

```diff
@@ -1,12 +1,37 @@
 page-waiting-room-to-car {
   ion-content > div {
     background-color: map-get($colors, "mes-white");
-    .mes-full-width-card {
+    .mes-validated-row {
```

Comment:
I've used the same CSS in my branch, but I changed the class name to `.mes-row` because the class can be used on rows where the field is read only or not being validated.

Reply:
...but I guess this might get refactored into a global style anyway

Reply:
have changed the name as it is a better indication.